### ISSUE TITLE
[Bug] The engine "node" is incompatible with this module. 

### ISSUE THREAD

**Issue by svengau** [fdaciuk/react-trianglify#11]
Hi, 
I use node 11.13.0 and when I install using "yarn add react-trianglify" I've got the following error:

```
The engine "node" is incompatible with this module. Expected version "=11.6.0". Got "11.13.0"
```

Please, could you update or even remove the engine node version ?

thanks,

**Comment by goldmont** [comment]
+1

**Comment by fdaciuk** [comment]
Fixed in `2.0.2` version :)
